Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -1076,18 +1129,16 @@ open class NavigationMapView: UIView {
         ]
         """
 
-        if let expressionData = expressionString.data(using: .utf8), let expJSONObject = try? JSONSerialization.jsonObject(with: expressionData, options: []) {
-
-            mapView.mapboxMap.__map.setStyleLayerPropertyForLayerId(NavigationMapView.routeDurationAnnotationsLayerIdentifier,
-                                                          property: "icon-anchor",
-                                                          value: expJSONObject)
-            mapView.mapboxMap.__map.setStyleLayerPropertyForLayerId(NavigationMapView.routeDurationAnnotationsLayerIdentifier,
-                                                          property: "text-anchor",
-                                                          value: expJSONObject)
+        if let expressionData = expressionString.data(using: .utf8),
+           let expJSONObject = try? JSONSerialization.jsonObject(with: expressionData, options: []) {
+            mapView.mapboxMap.__map.setStyleLayerPropertyForLayerId(routeDurationAnnotationsLayerIdentifier,
```

Comment:
#2996 will simplify a lot of the code in this method by using the map SDK’s usual runtime styling APIs.

Reply:
We should address and land #2996 as quickly as we can, but it is currently blocked by the Maps SDK.